modules/artifact/src/main/java/org/mule/module/artifact/classloader/ArtifactClassLoader.java

```diff
@@ -23,6 +25,18 @@
     URL findResource(String resource);
 
     /**
+     * Returns an enumeration of {@link java.net.URL <tt>URL</tt>} objects
+     * representing all the resources with the given name which are local
+     * to the classloader
+     *
+     * @param name The resource name
+     * @return An enumeration of {@link java.net.URL <tt>URL</tt>} objects for
+     * the resources
+     * @throws IOException If I/O errors occur
+     */
+    Enumeration<URL> findResources(final String name) throws IOException;
```

Comment:
This method is just a way to expose the same method used internally in the classloaders. Makes sense to use the same signature